src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Cloud tests from the CLI.md

```diff
@@ -319,16 +319,24 @@ export const options = {
   },
 };
 export default function () {
-  if (__ENV.LI_DISTRIBUTION === 'ashburnDistribution') {
+  if (__ENV.K6_CLOUDRUN_DISTRIBUTION === 'ashburnDistribution') {
     // do something
-  } else if (__ENV.LI_DISTRIBUTION == 'dublinDistribution') {
+  } else if (__ENV.K6_CLOUDRUN_DISTRIBUTION == 'dublinDistribution') {
     // do something
   }
 }
 ```
 
 </CodeGroup>
 
+<Blockquote mod="Attention" title="The LI_ prefix is deprecated">
+
+Previously, cloud environment variables were prefixed with `LI_` (for example, `LI_LOAD_ZONE`).
+These names are deprecated and will be removed in future versions of k6 Cloud.
```

Comment:
```suggestion
These names are deprecated and may be removed in future versions of k6 Cloud.
```

Not sure if we ever want to remove them :thinking: They are not really in the way of anything, so I doubt we will.